clients/v1/clients.proto

```diff
@@ -87,3 +87,5 @@ message SignOutSessionRequest {
 }
 
 message SignOutSessionResponse {}
+
+message ErrorRecordMismatch{}
```

Comment:
:P 

```suggestion
message ErrorRecordMismatch {}
```

Reply:
Docstrings would be helpful, both on what this message means, and where it's returned!